bin/dbtx-setup-dunedaq-from-recipe.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
```

Comment:
similarly, I would call the script `dbtx-create-workarea-from-receipe` (setup usually refers to environment)